src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,89 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Name;
+
+public class Tagging {
+	private static ArrayList<Tagging> tagList = new ArrayList<Tagging>();
+	private static ArrayList<String> taggingHistory = new ArrayList<String>();
```

Comment:
Reviewed in class, having taggingHistory within Tagging is an implementation of the other class diagram, not the one that is asked for